nemo/collections/asr/models/rnnt_bpe_models.py

```diff
@@ -183,6 +183,13 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
         )
         results.append(model)
 
+        model = PretrainedModelInfo(
+            pretrained_model_name="stt_ca_conformer_transducer_large",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_ca_conformer_transducer_large",
```

Comment:
Can you update the docs with these scores for Catalan language ?

Reply:
https://github.com/NVIDIA/NeMo/tree/main/docs/source/asr/data and https://github.com/NVIDIA/NeMo/blob/main/docs/source/asr/results.rst

Reply:
Just added~

Reply:
Scores are auto generated. Anyway it's fine I'll change if later